tests/pipelines/karras_ve/test_karras_ve.py

```diff
@@ -21,13 +21,11 @@
 from diffusers import KarrasVePipeline, KarrasVeScheduler, UNet2DModel
 from diffusers.utils.testing_utils import require_torch, slow, torch_device
 
-from ...test_pipelines_common import PipelineTesterMixin
-
 
 torch.backends.cuda.matmul.allow_tf32 = False
 
 
-class KarrasVePipelineFastTests(PipelineTesterMixin, unittest.TestCase):
+class KarrasVePipelineFastTests(unittest.TestCase):
```

Comment:
Guess this whole class needs an API update no? Could we maybe also open an issue for this?